deploy/example_catalog/cr-app-datafabric610.json

```diff
@@ -0,0 +1,232 @@
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "yarn-rm",
+        "label": {
+          "name": "ResourceManager"
+        },
+        "endpoint": {
+          "urlScheme": "https",
+          "path": "/cluster",
+          "port": 8090,
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "yarn-nm",
+        "label": {
+          "name": "NodeManager"
+        },
+        "endpoint": {
+          "urlScheme": "https",
+          "path": "/node",
+          "port": 8044,
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "yarn-hs",
+        "label": {
+          "name": "YARN HistoryServer"
+        },
+        "endpoint": {
+          "urlScheme": "https",
+          "path": "/jobhistory",
+          "port": 19890,
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "cldb",
+        "label": {
+          "name": "CLDB Web Port"
+        },
+        "endpoint": {
+          "port": 7443,
+          "isDashboard": false
+        }
+      },
+      {
+        "id": "mapr-client",
+        "label": {
+          "name": "Datafabric Client."
+        }
+      },
+      {
+        "id": "zookeeper",
+        "label": {
+          "name": "Zookeeper Server"
+        },
+        "endpoint": {
+          "port": 5181,
+          "isDashboard": false
+        }
+      },
+      {
+      "id": "oozie",
+      "label": {
+        "name": "OOZIE"
+      },
+      "endpoint": {
+        "urlScheme": "https",
+        "path": "/oozie",
+        "port": 11443,
+        "isDashboard": true
+      }
+    },
+      {
+        "id": "mysql",
+        "label": {
+          "name": "MySQL"
+        }
+      },
+      {
+        "id": "warden",
+        "label": {
+          "name": "Warden"
+        }
+      },
+      {
+        "id": "fileserver",
+        "label": {
+          "name": "FileServer"
+        }
+      },
+      {
+        "id": "ssh",
+        "label": {
+          "name": "ssh"
+        },
+        "endpoint": {
+          "port": 22,
+          "isDashboard": false
+        }
+      },
+      {
+        "id": "ticketserver",
+        "label": {
+          "name": "Simple Http server to distribute Datafabric tickets"
+        }
+      }
+    ],
+    "defaultImageRepoTag": "bluedata/datafabric610:1.0",
+    "defaultConfigPackage": {
+      "packageURL": "file:///opt/configscripts/appconfig-1.0.tgz"
+    },
+    "roles": [
+      {
+        "id": "control-system",
+        "cardinality": "1"
+      },
+      {
+        "id": "cldb",
+        "cardinality": "1"
+      },
+      {
+        "id": "oozie",
+        "cardinality": "0+"
+      },
+      {
+        "id": "edge",
+        "imageRepoTag": "bluedata/datafabric610edge:1.0",
+        "cardinality": "0+"
+      },
+      {
+        "id": "nodemanager",
+        "cardinality": "1+"
+      },
+      {
+        "id": "resource-manager",
+        "cardinality": "1+"
+      }
+    ]
```

Comment:
Is there a separate fileserver role? or just combined with cldb. Also we can't support HA for CLDB?

Reply:
there is no separate fileserver in this new design, fileserver runs on CLDB and nodemanager. CLDB HA needs license from MapR. I have added the license and so we can have CLDB HA, during conversation with Citi, they had expressed that they dont want to use that license. Hence i have kept the cardinality of CLDB to 1.